docs/docs/deploying-to-moovweb-xdn.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Deploying to the Moovweb XDN
+---
+
+This guide will cover deploying a Gatsby site to the Moovweb XDN.
+
+The [Moovweb XDN](https://www.moovweb.com/) is an all-in-one platform to develop, deploy, preview, experiment on, monitor, and run your headless frontend with a focus on support large. The XDN is focused on large, dynamic websites and best in class performance through an integrated CDN, CDN-as-JavaScript, predictive prefetching, and performance monitoring. Moovweb offers a free tier.
+
+This page summarizes the content in the XDN developer docs [Gatsby](https://developer.moovweb.com/guides/gatsby) guide. Refer to that guide for additional details.
+
+## Step 1: Sign up for a free XDN account
+
+You can sign up for free [here](https://moovweb.app/signup).
+
+## Step 2: Install the XDN CLI
+
+Install the XDN CLI via `npm`:
+
+```shell
+npm install -g @xdn/cli
+```
+
+## Step 3: Run XDN init
+
+Run the XDN `init` command in your project directory:
+
+```shell
+cd my-gatsby-project
+xdn init
+```
+
+This will automatically add all of the required XDN libraries and files to your project. The full list of changes XDN `init` performs is listed in the [Moovweb XDN Gatsby](https://developer.moovweb.com/guides/gatsby) guide.
+
+## Step 4: Test locally
+
+XDN CLI will emulate the XDN's integrated CDN locally. You can test the XDN integration with your project locally by using the `run` command:
+
+```shell
+xdn run
+```
+
+## Step 5: Deploy
+
+To deploy your app on the XDN run:
+
+```shell
+xdn deploy
+```
+
+When the deployment finishes, the output will confirm the final deployment URL. Below is an example:
```

Comment:
I tried following along with the steps in this guide, using just the default Gatsby starter as my project. (It's in the `moovweb-xdn-deploy-guide` directory.)

When I try to run this deploy command, I get the following webpack error:

```
success run page queries - 0.021s - 5/5 232.69/s
success write out requires - 0.004s
success Building production JavaScript and CSS bundles - 3.226s

 ERROR #98124  WEBPACK

Generating SSR bundle failed

Can't resolve '../../public/page-data/sq/d/1132682437.json' in '/Users/megansullivan/Documents/gatsby-what-gatsby/reviews/moovweb-xdn-deploy-guide/src/components'

If you're trying to use a package make sure that '../../public/page-data/sq/d/1132682437.json' is installed. If you're trying to use a local file make sure that the path is correct.

File: src/components/image.js

not finished Building static HTML for pages - 0.535s
```

Is there some other step that's needed to get this deploy to work? I'd like to figure out how to resolve this issue before we add this guide to the docs, so that we can prevent other users from running into similar problems.

Reply:
Here's a link to the repo I was testing with: https://github.com/meganesu/test-moovweb-xdn-deploy-guide

Reply:
That's odd. I just tried your repo and it worked for me. Here's a video: https://drive.google.com/file/d/1Ef07mfmDmQXpMeJSdLMoTb4ImHKCuR0Y/view?usp=sharing  

What node version are you running?

Reply:
We've reproduced the issue. It's due to a conflicting expectation in our code on how the public folder works (i.e. similar to https://github.com/gatsbyjs/gatsby/issues/529, we were expecting it to be cleanable). The result is that first deploy works (and any clean deploys) but incremental builds fail. We will add a clean step.

Reply:
Hi @ianand, thanks for looking into this. Let me know when the problem has been resolved and I'll be happy to give this PR another review!

Reply:
Will let you know when we've made the update. in the meantime, you can manually run an "npm run clean" to get your build working.

Reply:
@meganesu we pushed out a fix. you can start the tutorial over, or in your existing project do the following:

1. `rm -rf node_modules`
2. `rm package-lock.json`
3. `npm install`

And then `xdn deploy` should work.